docs/user/release_notes.md

```diff
@@ -8,6 +8,68 @@ Relevant links:
 - Project board: <https://github.com/orgs/Brewblox/projects/1>
 - Code repositories: <https://github.com/Brewblox>
 
+## Brewblox release 2024/03/??
+
+**firmware release 2024/03/??**
+
+**WARNING: Python 3.7 and Debian Buster are no longer supported. If you're using Debian Buster, see <https://www.brewblox.com/user/system_upgrades.html>**
+
+For this release, we have two new features: Sequences can use variables,
+and brewblox configuration are now generated based on settings in the new `brewblox.yml` file.
+
+## Variables
+
+Before, if you wanted to change a temperature value or a block name in a Sequence, you always had to update the sequence itself.
+If you had multiple instructions editing or waiting on the same block, you had to write the block name every time.
+
+Now, instruction arguments can either be a value (as before), or a reference to a variable.
+Variables are stored in the new *Variables* block. Multiple *Sequence* blocks can use the same *Variables* block.
+In the *Variables* block, you create a new variable, and give it a name and value.
+In the sequence instruction, you use a `$` + the variable name instead of an argument value.
+The variable type (block link, temperature, duration, etc) must match the argument type.
+
+Example syntax:
+
+```txt
+SET_SETPOINT target=$setpoint_block, setting=$beer_temp
+```
+
```

Comment:
Add:
An example use case for variables is to have a generic sequence that you don't change often, and a variable block that you update for each batch, with variables for setpoint settings and durations.